src/main/java/com/powsybl/openloadflow/sensi/DcSlowSensitivityAnalysis.java

```diff
@@ -0,0 +1,141 @@
+
+        LOGGER.info("Slow sensitivity analysis for contingency " + contingency.getId());
+
+        List<Equation> deactivatedEquations = new LinkedList<>();
+        List<EquationTerm> deactivatedEquationTerms = new LinkedList<>();
+        OpenSecurityAnalysis.deactivateEquations(createLfContingency(lfNetwork, contingency), equationSystem, deactivatedEquations, deactivatedEquationTerms);
+        contingency.getElements().forEach(element -> {
+            if (!element.getType().equals(ContingencyElementType.BRANCH)) {
+                throw new UnsupportedOperationException("Only contingency on a branch is yet supported");
+            }
+            LfBranch lfBranch = lfNetwork.getBranchById(element.getId());
+            connectivity.cut(lfBranch.getBus1(), lfBranch.getBus2());
+        });
+
+        // todo: Check what sensitivities still makes sense (still have a slack)
+        Map<SensitivityVariableConfiguration, SensitivityFactorGroup> factorsByVarConfig = indexFactorsByVariableConfig(network, connectivity, factors, lfNetwork, loadFlowParameters);
+        List<SensitivityValue> contingencyValues = analyse(lfNetwork, equationSystem, factorsByVarConfig, functionReferenceByBranch, loadFlowParameters);
+        OpenSecurityAnalysis.reactivateEquations(deactivatedEquations, deactivatedEquationTerms);
+        connectivity.reset();
+        return contingencyValues;
+    }
+
+    public List<SensitivityValue> analyse(LfNetwork lfNetwork, EquationSystem equationSystem, Map<SensitivityVariableConfiguration,
+            SensitivityFactorGroup> factorsByVarConfig, Map<String, Double> functionReferenceByBranch, LoadFlowParameters lfParameters) {
+        // initialize right hand side
+        DenseMatrix rhs = initRhs(lfNetwork, equationSystem, factorsByVarConfig);
+
+        // create jacobian matrix either using base network calculated voltages or nominal voltages
+        VoltageInitializer voltageInitializer = lfParameters.getVoltageInitMode() == LoadFlowParameters.VoltageInitMode.PREVIOUS_VALUES ? new PreviousValueVoltageInitializer()
+                : new UniformValueVoltageInitializer();
+        // FIXME: If the equation system represents two distinct connected component, the jacobian wont be invertible
+        // FIXME: We need to add a slack in each connected component
+        // FIXME: This is why in the previous version, we were reloading the network from scratch
+        JacobianMatrix j = createJacobianMatrix(equationSystem, voltageInitializer);
+
+        // solve system
+        DenseMatrix states = solveTransposed(rhs, j);
+
+        // calculate sensitivity values
+        return calculateSensitivityValues(lfNetwork, equationSystem, factorsByVarConfig, states, functionReferenceByBranch);
+    }
+
+    public Pair<List<SensitivityValue>, Map<String, List<SensitivityValue>>> analyse(Network network, List<SensitivityFactor> factors,
```

Comment:
This part is never used in your unit tests:
- I am not sure it is relevant to intoduce it in that PR or later ;
- Or we could make some unit tests that compare results from the fast and the slow approches.